stable/nginx-ingress/templates/controller-deployment.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
   name: {{ template "controller.fullname" . }}
+{{- if .Values.controller.rbac.enabled }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
This seems weird. I think we prefer to leave out the `namespace` in the metadata section of manifests and let helm take care of that when the `--namespace` flag is used.

However, you probably need it when you reference the `subject` in the role bindings, but it shouldn't be required here. Can you give it a try?